### ISSUE TITLE
Faraday gem is updated to a recent version

### ISSUE THREAD

**Issue by kfrz** [department-of-veterans-affairs/va.gov-team#2369]
## User Story or Problem Statement
We are on version `0.9.2` of the [Faraday](https://github.com/lostisland/faraday) library, but the latest version is `0.17.x`. We should upgrade to enable new features and have latest bugfixes.

## Goal
- [`faraday`](https://github.com/lostisland/faraday) gem is updated to a recent version

## Acceptance Criteria
- [x] `breakers` and `betamocks` will also have to be updated 
- [x] `faraday` is at the latest version
- [ ] Specs pass

## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by LindseySaari** [comment]
Currently waiting on permissions for breakers and betamocks

**Comment by kfrz** [comment]
Waiting on PRs to be merged into downstream repos.

**Comment by LindseySaari** [comment]
Still waiting on these two PR reviews:
https://github.com/department-of-veterans-affairs/breakers/pull/12

**Comment by annaswims** [comment]
I've approved https://github.com/department-of-veterans-affairs/breakers/pull/12 and 
https://github.com/department-of-veterans-affairs/betamocks/pull/17

Let me know if you need more access to merge.
